packages/winscp.vm/winscp.vm.nuspec

```diff
@@ -0,0 +1,12 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>winscp.vm</id>
```

Comment:
see https://community.chocolatey.org/packages/winscp?

Reply:
I agree, I was bringing over what was currently there, but it does make sense with the utilities to just use the community package. I think the tools should be dropped into the tools directory though. I'll remove the packages with community counterparts already.